extension/content/components/recipes/FilterObjects.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Divider, FormGroup, ControlLabel, Panel } from "rsuite";
+import { BrowserOptions } from "devtools/components/recipes/filters/BrowserOptions";
+import { GeoOptions } from "devtools/components/recipes/filters/GeoOptions";
+import SamplingOptions from "devtools/components/recipes/filters/SamplingOptions";
+
+export default function FilterObjects(props) {
+  const { filterObjectData, countryOptions, localeOptions } = props;
+  const filterValues = filterObjectData.reduce(
+    (obj, item) => Object.assign(obj, { [item.type]: item }),
+    {},
+  );
+
+  const handleFOChange = (key, value) => {
+    const pluralMapping = {
+      country: "countries",
+      locale: "locales",
+      channel: "channels",
+      version: "versions",
+    };
+    let changedFilterValues = filterObjectData.filter((entry) => {
```

Comment:
nit: this should be a `const`.

the only time we should use `let` is if we are reassigning. I think there's an eslint rule for this. I'll add it in to our eslint config.